Zhongli.Bot/Modules/Moderation/ModerationModule.cs

```diff
@@ -109,15 +112,54 @@ public async Task NoticeAsync(IGuildUser user, [Remainder] string? reason = null
         [Summary("Set a slowmode in the channel.")]
         [RequireBotPermission(ChannelPermission.ManageChannels)]
         [RequireUserPermission(ChannelPermission.ManageChannels)]
-        public async Task SlowmodeAsync(TimeSpan? length = null)
+        public async Task SlowmodeAsync(TimeSpan? length = null, ITextChannel? channel = null)
         {
-            var channel = (ITextChannel) Context.Channel;
-            var seconds = (int) (length ?? TimeSpan.Zero).TotalSeconds;
 
-            await channel.ModifyAsync(c => c.SlowModeInterval = seconds);
-            await Context.Message.AddReactionAsync(new Emoji("✅"));
+            if (length is null && channel is null)
+            {
+                var channels = Context.Guild.Channels.OfType<ITextChannel>().Where(c => c.SlowModeInterval is not 0);
+                var sb = new StringBuilder()
+                    .AppendLine("List of channels with slowmode active:")
+                    .AppendLine("--------------------------------------")
+                    .AppendLine("");
+
+                foreach (var cn in channels)
+                {
+
+                    sb.AppendLine($"{cn.Mention} => {cn.SlowModeInterval} seconds");
+                }
+
+                await ReplyAsync(sb.ToString());

```

Comment:
This should be an embed instead

And also, doing `.AppendLine()` works too if you don't want anything inside